tests/test_sft_trainer.py

```diff
@@ -754,6 +769,61 @@ def test_run_causallm_ft_pretokenized(dataset_path):
         assert "### Text: @NortonSupport Thanks much.\n\n### Label:" in output_inference
 
 
+@pytest.mark.parametrize(
+    "datafiles, datasetconfigname",
+    [
+        (
+            [
+                TWITTER_COMPLAINTS_DATA_INPUT_OUTPUT_JSONL,
+                TWITTER_COMPLAINTS_DATA_INPUT_OUTPUT_JSONL,
+            ],
```

Comment:
Would there be a benefit in using multiple types of data here such as JSONL for one dataset, Arrow for another, and Parquet for a third?

Reply:
May not be necessary, Good to have it though. Added!

Reply:
Has this been added? I do not see a mix of the different type of data @willmj asked here.

Reply:
Yes this was added. I can see it in commit changes [here](https://github.com/foundation-model-stack/fms-hf-tuning/pull/412/files#diff-406c3c4d02c1ff158009b04d26a6b2d357f05d11e8c35c3f2ddfc1b54649a022). Let me know of other changes required in this unit test.